### PR TITLE
Fix  lead time and prep time in dine in fulfillment

### PR DESCRIPTION
**Pull request by jayant13o6**
## Description
Resolve issues: #987 and #984 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Changes and Fixes
- SCHEDULED_DINEIN replaced with PREODER_DINEIN
- dineInLeadTime and dineInPrepTime columns added in timeSlot table
- New field added in UI form for Slot Interval 

## Screenshots 
(prefer all screen sizes images)
image

## Checklist
- [x] Database schema is updated
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

Affected Apps
- [ ] Admin
    - [x] Menu>>Recurrences
- [ ] Server
    - [ ] Payment
    - [ ] Email
    - [ ] SMS
    - [ ] Notifications
    - [ ] Integration
    - [ ] Hasura auth
- [ ] Template Engine

### CODE REVIEW

File: admin/src/apps/carts/views/forms/ondemand/tunnels/Fulfillment/index.jsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ const Content = ({ panel }) => {
             orderTab =>
                orderTab.orderFulfillmentTypeLabel === 'PREORDER_DINEIN' ||
                orderTab.orderFulfillmentTypeLabel === 'ONDEMAND_DINEIN' ||
-               orderTab.orderFulfillmentTypeLabel === 'SCHEDULED_DINEIN'
+               orderTab.orderFulfillmentTypeLabel === 'PREORDER_DINEIN'
```
Comment (ng11tk) [inline admin/src/apps/carts/views/forms/ondemand/tunnels/Fulfillment/index.jsx:85]: line no. 83 is same as this line

**Review by ng11tk — CHANGES_REQUESTED** [review]
see the comment for changes

**Review by ng11tk — APPROVED** [review]

File: admin/src/apps/menu/views/Forms/Recurrences/Main/tunnels/TimeSlotTunnel/index.jsx [hunk 2]
```diff
@@ -50,6 +50,14 @@ const TimeSlotTunnel = ({ closeTunnel }) => {
          errors: [],
       },
    })
+   const [slotInterval, setSlotInterval] = React.useState({
+      value: '00:15',
```
Comment (rahul72925) [inline admin/src/apps/menu/views/Forms/Recurrences/Main/tunnels/TimeSlotTunnel/index.jsx:54]: slot interval format is HH:mm:ss.

Comment (jayant13o6) [reply admin/src/apps/menu/views/Forms/Recurrences/Main/tunnels/TimeSlotTunnel/index.jsx:54]: yes, it is 15 mins not 15 secs

**Pull request merged**